clusterman/migration/worker.py

```diff
@@ -131,4 +157,44 @@ def event_migration_worker(migration_event: MigrationEvent, worker_setup: Worker
     :param MigrationEvent migration_event: event instance
     :param WorkerSetup worker_setup: migration setup
     """
-    pass  # TODO
+    prescaling_applied = False
+    manager = PoolManager(migration_event.cluster, migration_event.pool, SUPPORTED_POOL_SCHEDULER, fetch_state=False)
+    connector = cast(KubernetesClusterConnector, manager.cluster_connector)
+    connector.set_label_selectors(migration_event.label_selectors, add_to_existing=True)
+    manager.reload_state()
+    try:
+        if worker_setup.disable_autoscaling:
+            logger.info(f"Disabling autoscaling for {migration_event.cluster}:{migration_event.pool}")
+            disable_autoscaling(
+                migration_event.cluster,
+                migration_event.pool,
+                SUPPORTED_POOL_SCHEDULER,
+                time.time() + worker_setup.expected_duration,
+            )
+        if worker_setup.prescaling:
+            nodes = manager.get_node_metadatas(AWS_RUNNING_STATES)
+            offset = worker_setup.prescaling.of(len(nodes))
+            avg_weight = mean(node.instance.weight for node in nodes)
+            original_capacity = manager.target_capacity
+            prescaled_capacity = round(manager.target_capacity + offset * avg_weight)
```

Comment:
feel free to drop: should we parenthesize the math here or just trust that people will read this according to the precedence rules?

Reply:
Will add some parenthesis, never trust math.